setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name = 'OpenFisca-Core',
-    version = '4.2.1',
+    version = '4.2.2',
```

Comment:
The goal of this PR was to fix a bug when neutralizing a updating a column to make sure the set_input and default value are correctly transmitted to the update variable.